frontend/src/utils/action.tsx

```diff
@@ -77,3 +78,18 @@ export const RemoveFromRateLater = (asyncCallback?: () => void) => {
   };
   return RemoveFromRateLaterComponnent;
 };
+
+export const AddDetailedConsultation = ({ uid }: { uid: string }) => {
+  const { t } = useTranslation();
+  return (
+    <Tooltip title={`${t('actions.detailed')}`} placement="left">
```

Comment:
<img width="253" alt="Screenshot 2022-04-20 at 08 03 51" src="https://user-images.githubusercontent.com/8818081/164161268-767f3de4-0fdb-442f-8091-0d0653a1e37c.png">

The text of the tooltip is missing. You need to add it in the files `translation.json` to add text for French and English

Reply:
fixed by https://github.com/tournesol-app/tournesol/pull/827/commits/72fb63bd077f1c021b54efa492b27463d9323545